rewrite.go

```diff
@@ -109,7 +109,14 @@ func rewriteGoFile(name, qual string, paths []string) error {
 // sep is the signature set of path elements that
 // precede the original path of an imported package
 // in a rewritten import path.
-const sep = "/Godeps/_workspace/src/"
+var sep = "/Godeps/_workspace/src/"
+var VendorExperiment = os.Getenv("GO15VENDOREXPERIMENT") == "1"
+
+func init() {
```

Comment:
Hmm. I likened it to flag handling, which is done elsewhere in init functions.

Are you suggesting that init should call `defaultSep(VendorExperiment)` to set sep or that everywhere sep is currently used it should be replaced with `defaultSep(VendorExperiment)` ?